pilot/pkg/bootstrap/server.go

```diff
@@ -296,6 +297,17 @@ func checkForMock(registries []string) bool {
 	return false
 }
 
+// Check if Kubernetes registry exists in PilotArgs's Registries
+func checkForKubernetes(registries []string) bool {
+	for _, r := range registries {
+		if strings.ToLower(r) == "kubernetes" {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
Check line 404 please. Since we added this helper method, code checking if k8s is in the registries should use this one.

Reply:
if you are referring to `func (s *Server) initKubeClient(args *PilotArgs)`,  it seems it just initializes kube client.  I could check if kubeclient was initialized in Server, then it runs in kubernetes environment, but it does not look straight forward.  On the other hand this tiny function  is straight forward  in what its purpose is.